components/collapsible/collapsible_lazy_show.vue

```diff
@@ -101,8 +101,6 @@ export default {
         if (!element.style.height) {
           element.style.height = '0px';
         }
-
-        element.style.display = null;
```

Comment:
I removed this because it does nothing, the display property is not controlled in the transition.